.circleci/config.yml

```diff
@@ -7,9 +7,18 @@ executors:
     parameters:
       image:
         type: string
+        # First 10.x LTS release, but old Yarn
```

Comment:
Yeah, that was one of the reasons we put yarn binary in the repo (to avoid updating old yarn available in container on each run)